packages/embark-plugin-cmd/src/index.js

```diff
@@ -1,4 +1,4 @@
-const { runCmd } = require('embark-utils');
+const utils = require('embark-utils');
```

Comment:
Is there a reason why you changed that? I think I prefer the old version.

Reply:
Must have been from updating master with a stash.